packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
For the css module styling, this file needs updated to be locally scoped. [Here](https://github.com/cerner/terra-core/blob/master/packages/terra-button-group/src/ButtonGroup.scss) is a simple example of what your styling might look like. 

Our team has decided to use css styling syntax for class names, so `.cardContainer` should be changed to `.card-container`.